packages/harpguru-core/src/components/harp-cell/harp-cell.test.tsx

```diff
@@ -20,6 +21,7 @@ test('A component is rendered with the Degree or Pitch value in its text view de
     if (stateItem === 'activeExperienceMode') return [ExperienceModes.Explore]
     if (stateItem === 'quizQuestion') return [DegreeIds.Root]
     if (stateItem === 'activeDisplayMode') return [DisplayModes.Degree]
+    if (stateItem === 'lockedCovariant') return [CovariantMembers.HarpKey]
```

Comment:
This is getting a lot. We really do need a common mock object for global state which can be set up easily.

Reply:
#21 raised